packages/webapp/pages/squads/discover/[id].tsx

```diff
@@ -0,0 +1,83 @@
+import { oneHour } from '@dailydotdev/shared/src/lib/dateFormat';
+import { GetServerSidePropsContext, GetServerSidePropsResult } from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import React, { ReactElement } from 'react';
+import {
+  getSourceCategory,
+  SourceCategory,
+} from '@dailydotdev/shared/src/graphql/source/categories';
+import { useSources } from '@dailydotdev/shared/src/hooks/source/useSources';
+import InfiniteScrolling, {
+  checkFetchMore,
+} from '@dailydotdev/shared/src/components/containers/InfiniteScrolling';
+import { UnfeaturedSquadGrid } from '@dailydotdev/shared/src/components/cards/squad/UnfeaturedSquadGrid';
+import { Squad } from '@dailydotdev/shared/src/graphql/sources';
+import { getLayout } from '../../../components/layouts/FeedLayout';
+import { mainFeedLayoutProps } from '../../../components/layouts/MainFeedPage';
+
+interface SquadCategoryPageProps {
+  category: SourceCategory;
+}
+
+function SquadCategoryPage({ category }: SquadCategoryPageProps): ReactElement {
+  const { result } = useSources({
+    query: { categoryId: category.id, isPublic: true },
+  });
+  const flatSources =
+    result.data?.pages.flatMap((page) => page.sources.edges) ?? [];
+
+  return (
+    <div className="mx-auto mt-5 flex w-full max-w-[99.75rem] flex-col px-5">
```

Comment:
What's this max w 👀

Reply:
Oh, I initially thought we would need some max-w, but looks like we actually don't. Should update this.

![Screenshot 2024-09-10 at 12 02 29 AM](https://github.com/user-attachments/assets/79adf409-9c2c-468d-9de7-d920ce2b6eb6)